tutorials/run-laravel-on-appengine-flexible/index.md

```diff
@@ -0,0 +1,161 @@
+locally.
+
+1. Use the Cloud SDK from command-line to run the following command. Copy the
+`connectionName` value for the next step.
+  ```sh
+  gcloud beta sql instances describe YOUR_INSTANCE_NAME
+  ```
+
+1. Start the Cloud SQL Proxy using the connection name from the previous step:
+  ```sh
+  cloud_sql_proxy -instances=YOUR_INSTANCE_CONNECTION_NAME=tcp:3306
+  ```
+
+1. Use the MySQL client or similar program to connect to your instance and
+  create a database for the application. When prompted, use the root password
+  you configured.
+  ```sh
+  mysql -h 127.0.0.1 -u root -p -e "CREATE DATABASE laravel;"
+  ```
+
+1. Run the database migrations for Laravel. This can be done by setting your
+  parameters in `.env` or by passing them in as environemnt variables. Be sure
+  to replace `YOUR_DB_PASSWORD` below with the root password you configured:
+  ```sh
+  # create a migration for the session table
+  php artisan session:table
+  DB_DATABASE=laravel DB_USERNAME=root DB_PASSWORD=YOUR_DB_PASSWORD php artisan migrate --force
+  ```
+
+1. Edit `config/database.php` and add a line for "unix_socket" to the
+  'mysql' connection configuration:
+  ```php
+  'mysql' => [
+      // ...
+      'unix_socket' => env('DB_SOCKET', ''),
+  ```
+
+1. Modify your `app.yaml` file with the following contents:
+
+        runtime: php
+        env: flex
+
+        runtime_config:
+          document_root: public
+
+        # required on some platforms so ".env" is not skipped
+        skip_files: false
+
+        env_variables:
+          # The values here will override those in ".env". This is useful for
+          # production-specific configuration. However, feel free to set these values
+          # in ".env" instead if you prefer.
+          APP_LOG: errorlog
+          STORAGE_DIR: /tmp
+          CACHE_DRIVER: database
+          SESSION_DRIVER: database
+          ## Set these environment variables according to your CloudSQL configuration.
+          DB_HOST: localhost
+          DB_DATABASE: laravel
+          DB_USERNAME: root
+          DB_PASSWORD: YOUR_DB_PASSWORD
+          DB_SOCKET: /cloudsql/YOUR_CLOUDSQL_CONNECTION_NAME
+
+        beta_settings:
+            # for Cloud SQL, set this value to the Cloud SQL connection name,
+            # e.g. "project:region:cloudsql-instance"
+            cloud_sql_instances: "YOUR_CLOUDSQL_CONNECTION_NAME"
+
+1. Replace each instance of `YOUR_DB_PASSWORD` and `YOUR_CLOUDSQL_CONNECTION_NAME`
+with the appropriate you created for your CloudSQL instance above.
+
+[php-gcp]: https://cloud.google.com/php
+[laravel]: http://laravel.com
+[laravel-install]: https://laravel.com/docs/5.4/installation
+[laravel-welcome]: https://storage.googleapis.com/gcp-community/tutorials/run-laravel-on-appengine-flexible/welcome-page.png
+[composer-json]: https://storage.googleapis.com/gcp-community/tutorials/run-laravel-on-appengine-flexible/composer-json.png
+[cloudsql-create]: https://cloud.google.com/sql/docs/create-instance
```

Comment:
Link should point to either https://cloud.google.com/sql/docs/mysql/create-instance or https://cloud.google.com/sql/docs/postgres/create-instance.